underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -83,26 +83,24 @@
    * @return the created {@link GCSUnderFileSystem} instance
    * @throws ServiceException when a connection to GCS could not be created
    */
-  public static GCSUnderFileSystem createInstance(AlluxioURI uri, Map<String, String> ufsConf)
-      throws ServiceException {
+  public static GCSUnderFileSystem createInstance(
+      AlluxioURI uri, UnderFileSystemConfiguration ufsConf) throws ServiceException {
```

Comment:
Should these just be called `conf` since the type makes it clear (applies elsewhere as well)?